tomcat/src/main/java/org/dogtagpki/jss/tomcat/Http11NioProtocol.java

```diff
@@ -73,12 +73,12 @@ public void setServerCertNickFile(String serverCertNickFile) {
         tomcatjss.setServerCertNickFile(serverCertNickFile);
     }
 
-    public boolean getEnabledOCSP() {
-        return tomcatjss.getEnableOCSP();
+    public boolean getEnableRevocationCheck() {
+        return tomcatjss.getEnableRevocationCheck();
     }
 
-    public void setEnableOCSP(boolean enableOCSP) {
-        tomcatjss.setEnableOCSP(enableOCSP);
+    public void setEnableRevocationCheck(boolean enableRevocationCheck) {
+        tomcatjss.setEnableRevocationCheck(enableRevocationCheck);
```

Comment:
As discussed on Slack, we should keep the `enabledOCSP` param for backward compatibility, but now the getter/setter for that param can call `TomcatJSS.enableRevocationCheck`'s getter/setter instead.